prestodb/centos6-oj8/files/usr/local/bin/apply-diffs

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
This file is unused

Reply:
in this commit -- yes. It's used in Dockerfiles of images that depend on this image